src/sql/workbench/services/notebook/browser/sql/sqlSessionManager.ts

```diff
@@ -620,6 +624,7 @@ export class SQLFuture extends Disposable implements FutureInternal {
 
 	private convertToError(msg: IResultMessage | string): nb.IIOPubMessage {
 		if (msg) {
+			this._errorOccurred = true;
```

Comment:
Shouldn't this be outside the loop? We know the message is an error if this is called? Or why are we testing for null, we know all locations this is called from the msg must exist so can avoid this confusion.